python/mxnet/symbol.py

```diff
@@ -48,6 +48,10 @@ def __repr__(self):
         return '<%s %s>' % (self.__class__.__name__,
                             'Grouped' if name is None else name)
 
+    def __iter__(self):
```

Comment:
This is a pretty fundamental change.  Maybe this should be a named method like "outputs"?